src/codegen/signal_body.rs

```diff
@@ -27,6 +28,11 @@ impl Builder {
         self
     }
 
+    pub fn is_detailed(&mut self, value: bool) -> &mut Builder {
```

Comment:
That's a weird name for a setter... What about `detailed` or something along the line?